apps/dashboard/src/components/subscribers/subscriber-row.tsx

```diff
@@ -165,7 +174,7 @@ export const SubscriberRow = ({ subscriber }: SubscriberRowProps) => {
                 <DropdownMenuItem
                   className="text-destructive cursor-pointer"
                   onClick={() => {
-                    setIsDeleteModalOpen(true);
+                    setTimeout(() => setIsDeleteModalOpen(true), 0);
```

Comment:
Without this fix, the modal breaks the dropdown menu from opening again. This might be because the dropdown works like a modal, but I didn't investigate that deep.